src/App.tsx

```diff
@@ -1,223 +1,220 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import {
+  addTodos,
+  deleteTodo,
+  getTodos,
+  patchTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/Page/ErrorNotification';
+import { FilterTodos } from './components/Page/FilterTodo';
+import { Header } from './components/Page/Header';
```

Comment:
It is noy a good name for this component. Reread [Task checklist point 3](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md) and rename. Component names examples are there more meaningful.